### ISSUE TITLE
Manually trigger field validation

### ISSUE THREAD

**Issue by maakit** [1000hz/bootstrap-validator#359]
Is there a method to manually trigger field validation.

I have a custom rule which validates a select **A** based on its value and the value of another select **B**.
If I change select **B** I want to trigger the validation for select **A**.
Something like `<select id="B" onchange="$('#A').validate();">`

**Comment by 1000hz** [comment]
trigger an `input` event on it. 
```js
$('#b').trigger('input')
```

**Comment by maakit** [comment]
I tried this solution using blur, input and change events (using jQuery 1.12.1) but it doesn't fire the field validation in my setup(tested FF and Chrome).
Ugly code I am using meanwhile and what dows work is:
`var orival = $('#A').val();
$('#A').val(orival=='1'?'0':'1').trigger('blur');
$('#A').val(orival).trigger('blur');`

Element #A can be either 0 or 1
Get current value (orival)
Set select to other value and trigger blur event.
Set select to original value and trigger blur event.

**Comment by colgreen** [comment]
Can we re-open this please? The suggested approach doesn't work:

   $('#A').trigger('input')

I suspect that this doesn't trigger validation because the input field's value has not changed, hence maakit's approach of changing the value to something else and back again, which could of course cause problems elsewhere, e.g. if some code reacts to the temp value being changed to.

**Comment by 1000hz** [comment]
I think you might be running an old version of the validator. For a little while, the validator wasn't validating fields if their values hadn't changed.
